### ISSUE TITLE
Dashboard should indicate teams that I am a member of vs exposed pipelines

### ISSUE THREAD

**Issue by jama-pivotal** [concourse/concourse#2427]
With the introduction of users in 4.0.0 and the new user based auth flow, I find that I am looking at a lot of pipelines in the Concourse dashboard. Some of them are pipelines on teams that I belong to, but others are from publicly exposed pipelines. I would like to have the dashboard prioritize the listing of teams based on teams I have membership into. I'm thinking of two options

1. Just sort the list so that teams I am members of show up first alphabetically, then show me the public teams and pipelines alphabetically
2. Some visual indicator on teams for teams I'm a member of, or on public pipelines

cc @Lindsayauchin for comment

**Comment by Lindsayauchin** [comment]
![screen shot 2018-07-30 at 11 33 08 am](https://user-images.githubusercontent.com/27746614/43407321-6ae96892-93ec-11e8-9851-d736e8895e4b.png)

**Comment by pivotal-saman-alvi** [comment]
The token returns the teams the user is authorized to view for the `/get` endpoint on teams, so there is no current way to differentiate between a team you belong to vs. a public team. There is a way to differentiate between private / public pipelines for a team, although that does not seem as beneficial? @jama-pivotal, thoughts?  

This change would also mean the current method of ordering pipelines by build status would be removed in favor of alphabetical ordering.

**Comment by jama-pivotal** [comment]
j/k it sounds like you can distinguish teams you are part of vs exposed pipeline :)

**Comment by mhuangpivotal** [comment]
There are actually four different types of teams:

- `main` team
- teams with auth that I am a member of
- teams with auth that I am not a member of, but have exposed pipelines
- public teams

Do we want to combine the last two cases and considered them "public"?

**Comment by Lindsayauchin** [comment]
Updated the design as per the conversation to add a lable instead of an icon. 
Updated to 3 categories of teams: Public, private, exposed. 

![screen shot 2018-08-24 at 12 10 04 pm](https://user-images.githubusercontent.com/27746614/44595517-366fd880-a797-11e8-96c2-5de06ff76282.png)

![screen shot 2018-08-24 at 12 08 40 pm](https://user-images.githubusercontent.com/27746614/44595522-396ac900-a797-11e8-883a-50cc1fc7a0af.png)

**Comment by jama-pivotal** [comment]
We had a discussion locally with the team and we decided that we should experiment with just two categories instead: Member and Exposed

To be clear, this is the breakdown of concepts we were talking about:

**Public: ** A pipeline that has "allow all users" to view, edit and access
**Member:** A pipeline that i'm a member of via the pipeline's auth settings configured on `set-team`
**Exposed:** A pipeline that I'm not a member of but I can see because of `fly expose pipeline` https://concourse-ci.org/managing-pipelines.html#fly-expose-pipeline

Points of confusion:
* do people really consider pipelines that have allow all users as "public"? This is potentially confusing with the `public` label on jobs: https://concourse-ci.org/jobs.html#job-public  
* In the case of pipelines where it has no auth, do people see it as automatically being a member of that team?

Decision:
After some discussion with the team, we decided to keep the first version simple with two labels:
* Member
* Exposed

If we deploy this simple change to our internal environment (Wings) we can interview users and see how they feel about this separation!
